dashboard/server.R

```diff
@@ -0,0 +1,78 @@
+
```

Comment:
How much data is available for the SMR audits? If we only have a couple of years, I'm more inclined to recommend removing the filter for that and showing all the data available over columns?

Reply:
Its a relatively small dataset, right now we have just over 1.5K rows, which we'll update on the repo after merging this branch. We have 3 audits left to add to the dataset, so I expect we'll end up wit just about 2k rows to display :)